components/body/desktop.styl

```diff
@@ -0,0 +1,116 @@
+@media (min-device-width 1200px)
+  .main-side-margin
+    margin 0 170px
+
+  .body-content-container
+    padding-top 0px
+
+  #section1.body-section-content, .section-three .body-section-content, .section-four .body-section-content, .body-section-five-content.js-section-content
+    display flex
+    -webkit-flex-direction row
+    flex-direction row
+
+  .body-section-five-content
+    display block
+
+  .section-two .body-section-content
+    display flex
+    flex-direction row-reverse
+
+  .carousel-track__image
```

Comment:
Probably makes more sense to include this stuff in the carousel stylesheet and do a different breakpoint there.